lucien-ios/Lending/LendABookViewController.swift

```diff
@@ -0,0 +1,157 @@
+//
+//  LendABookViewController.swift
+//  lucien-ios
+//
+//  Created by Ahmed, Guled on 10/19/17.
+//  Copyright © 2017 Intrepid Pursuits. All rights reserved.
+//
+
+import UIKit
+import RxSwift
+import RxCocoa
+
+class LendABookViewController: UIViewController, UITableViewDelegate, AlertDisplaying {
+
+    // MARK: - Private IBOutlets
+
+    @IBOutlet private weak var comicPhoto: UIImageView!
+    @IBOutlet private weak var seriesTitleLabel: UILabel!
+    @IBOutlet private weak var storyTitleLabel: UILabel!
+    @IBOutlet private weak var volumeLabel: UILabel!
+    @IBOutlet private weak var tableView: UITableView!
+    @IBOutlet private weak var comicInfoView: UIView!
+
+    // MARK: - Private Constants
+
+    private let disposeBag = DisposeBag()
+    private let viewModel = LendingBookViewModel()
+
+    // MARK: - Private Variables
+
+    private var nextButton = UIBarButtonItem()
+    private let lendingCell = UINib(nibName: "LendingTableViewCell", bundle: nil)
+    private let headerView = UIView()
+
+    // MARK: - Variables
+
+    var selectedIndex: Int?
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureNavigationController()
+        configureTableView()
+
+        viewModel.getAllUsers { [weak self] (users, error) in
+            if error == nil {
+                guard let users = users else { return }
+                self?.viewModel.users.value = users
+                DispatchQueue.main.async {
+                    self?.tableView.reloadData()
+                }
+            } else {
+                self?.showAlert(title: "Error", message: "Our service is currently encountering an issue. Please ensure that you are connected to the internet and try again.")
+            }
+        }
+    }
+
+    // MARK: - Private Instance Methods
+
+    private func configureTableView() {
+        tableView.register(lendingCell, forCellReuseIdentifier: "userCell")
+
+        tableView.rx.itemSelected.subscribe(onNext: { [weak self] indexPath in
+            self?.nextButton.isEnabled = true
+            self?.nextButton.tintColor = LucienTheme.dark
+            self?.selectedIndex = indexPath.row
+            self?.tableView.reloadData()
+        }) >>> disposeBag
+
+        viewModel.users.asObservable().bind(to: tableView.rx.items(cellIdentifier: "userCell",
```

Comment:
put `cellIdentifier` on its own line indented one tab from `viewModel` and adjust the other lines' indentations to match. Also put the `) {` on its own line so it's not indented so far